.devcontainer/docker-compose.yaml

```diff
@@ -1,6 +1,20 @@
 version: "3.8"
 
 services:
+  apollo-router:
+    image: ghcr.io/apollographql/router:v1.41.0-rc.2
+    volumes:
+      - ./supergraph.graphql:/dist/schema/supergraph.graphql
+      - ./router.yaml:/dist/config/router.yaml
+    ports:
+      - 4001:4000
+    command:
+      - --dev
+      - -config
+      - config/router.yaml
```

Comment:
I might be tempted to use absolute paths here as the fact the binary runs from `dist` is a piece of context someone may not have